src/views/catalog/customize/components/CustomizeSource/quantity.ts

```diff
@@ -0,0 +1,63 @@
+import byteSize, { ByteSizeResult } from 'byte-size';
+
+const multipliers: Record<string, number> = {};
```

Comment:
@avivtur didn't you implement this already in the disks shared files?
we need a share util for units, sizes

Reply:
we are currently in the phase of getting things done for each plugin extension,
it means that @avivtur may have his own version of the same functionlity,

once we have a working extension, we may go over it, choose the best implementation and move it to a shared directory